RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -0,0 +1,289 @@
+                    {
+                        var utxo = new JObject();
+                        utxo["txid"] = txId;
+                        utxo["n"] = unspent.Key;
+                        utxo["value"] = (double) (decimal) unspent.Value;
+                        total += unspent.Value;
+
+                        unspents.Add(utxo);
+                        if (unspents.Count > _rpcMaxUnspents)
+                            return (unspents, total);
+                    }
+                }
+                return (unspents, total);
+            }
+
+            JObject json = new JObject();
+            JArray balances = new JArray();
+            json["balance"] = balances;
+            json["address"] = scriptHash.ToAddress();
+            for (byte tokenIndex = startingToken; maxIterations-- > 0; tokenIndex++)
+            {
+                byte[] prefix = new [] { tokenIndex }.Concat(scriptHash.ToArray()).ToArray();
+
+                var (unspents, total) = RetreiveUnspentsForPrefix(prefix);
+
+                if (unspents.Count <= 0) continue;
+
+                var balance = new JObject();
+                balance["unspent"] = unspents;
+                balance["asset_hash"] = nativeAssetIds[tokenIndex].ToString().Substring(2);
+                balance["asset_symbol"] = balance["asset"] = nativeAssetNames[tokenIndex];
+                balance["amount"] = new JNumber((double) (decimal) total); ;
+                balances.Add(balance);
+            }
+
+            return json;
+        }
+
+        private JObject ProcessGetUnclaimedSpents(UInt160 scriptHash)
+        {
+            var dbCache = new DbCache<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs>(
+                _db, null, null, SystemAssetSpentUnclaimedCoinsPrefix);
+            return GenerateUtxoResponse(scriptHash, 1, 1, dbCache);
+        }
+
+        private JObject ProcessGetUnspents(JArray _params)
+        {
+            UInt160 scriptHash = GetScriptHashFromParam(_params[0].AsString());
+            byte startingToken = 0; // 0 = Utility Token (GAS), 1 = Governing Token (NEO)
+            int maxIterations = 2;
+
+            if (_params.Count > 1)
+            {
+                maxIterations = 1;
+                bool isGoverningToken = _params[1].AsBoolean();
+                if (isGoverningToken) startingToken = 1;
+            }
+
+            var unspentsCache = new DbCache<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs>(
+                _db, null, null, SystemAssetUnspentCoinsPrefix);
+
+            return GenerateUtxoResponse(scriptHash, startingToken, maxIterations, unspentsCache);
+        }
+
+        public JObject OnProcess(HttpContext context, string method, JArray _params)
+        {
+            if (_shouldTrackUnclaimed && method == "getunclaimedspents")
+                return ProcessGetUnclaimedSpents(GetScriptHashFromParam(_params[0].AsString()));
```

Comment:
I expect to adjust this API to get available claim transactions. I just did this quickly intermediately.

Reply:
This PR is of great impact for the ecosystem and applications in the upper layer.
Congrulations.

Reply:
Output should match neoscan get_claimable api results.

Reply:
I implemented the getclaimable rpc call as mentioned.